lib/plugins/aws/package/compile/events/apiGateway/index.js

```diff
@@ -296,7 +296,7 @@ class AwsCompileApigEvents {
       },
```

Comment:
Shouldn't `after:deploy:deploy` hook also be marked as async? I believe it's calling a function that returns a promise